manager/info.sh

```diff
@@ -34,8 +36,10 @@ fi
 eksctl utils write-kubeconfig --cluster=$CORTEX_CLUSTER_NAME --region=$CORTEX_REGION | grep -v "saved kubeconfig as" | grep -v "using region" | grep -v "eksctl version" || true
 
 operator_endpoint=$(get_operator_endpoint)
-apis_endpoint=$(get_apis_endpoint)
+api_load_balancer_endpoint=$(get_api_load_balancer_endpoint)
+api_gateway_endpoint=$(get_api_gateway_endpoint)
 
 # before modifying this, search for this prefix
-echo "operator endpoint: $operator_endpoint"
-echo "apis endpoint:     $apis_endpoint"
+echo "operator endpoint:          $operator_endpoint"
+echo "api load balancer endpoint: $api_load_balancer_endpoint"
+echo "api gateway endpoint:       $api_gateway_endpoint"
```

Comment:
add newline `echo ""`
to have at least one new line to next output `fetching cluster status ...`